plugins/locale.js

```diff
@@ -1,14 +1,10 @@
 import { join } from "node:path";
 import fs from "node:fs";
 import fp from "fastify-plugin";
-import PathResolver from "../lib/path.js";
 import chalk from "chalk";
 
-export default fp(async function locale(fastify, { cwd = process.cwd(), locale = "en", development }) {
-  const resolver = new PathResolver({ development, cwd });
-
-  const contentFile = await resolver.resolve("./content");
-  const compiledFileExtension = contentFile.typescript ? ".ts" : ".mjs";
+export default fp(async function locale(fastify, { cwd = process.cwd(), locale = "en" }) {
+  const compiledFileExtension = ".mjs";
```

Comment:
Are we unable to use .js here? I think I remember a discussion about this before but don't remember what the outcome was.

Reply:
> Are we unable to use .js here? I think I remember a discussion about this before but don't remember what the outcome was.

Would require a change in lingui itself. Absolutely possible, I'll check out how easy it is to make a PR there